app/components/photo-tags/photo-tags.js

```
@@ -8,25 +8,54 @@ import { htmlSafe } from '@ember/template';
+    }
   }
 
   @action
   toggleShowTags() {
     this.showTags = !this.showTags;
   }
 
+  @action
+  toggleIncludeOldMembers() {
+    const newValue = !this.includeOldMembers;
+    this.router.transitionTo({ queryParams: { includeOldMembers: newValue ? 'true' : 'false' } });
```

ESLint

[error] Replace `·queryParams:·{·includeOldMembers:·newValue·?·'true'·:·'false'·}` with `⏎······queryParams:·{·includeOldMembers:·newValue·?·'true'·:·'false'·},⏎···`

(prettier/prettier)